Fitness/Models/Habit.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Habit.swift
+//  Fitness
+//
+//  Created by Joseph Fulgieri on 3/20/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import Foundation
+
+
+// Habits are stored by category as a dictionary mapping habit titles to an array of
+// dates on which that habit was checked in [dates are strings of form "MMDDYYYY"].
+// The active habits are stored in an array of form: [ cardio habit, strength habit, mindfulness habit]
+struct Habit {
+    let title: String
+    let dates: [Date]
+    
+    static func getActiveHabitTitle(type: HabitTrackingType) -> String? {
+        let defaults = UserDefaults.standard
+        
+        guard let activeHabits = defaults.stringArray(forKey: Identifiers.activeHabits) else { return nil }
+        
+        if activeHabits.count < 3 || activeHabits[type.rawValue] == "" {
+            return nil
```

Comment:
Why are we returning nil here?

Reply:
Since there might not always be a selected habit, I made the function return an optional, and when either of these conditions are true there's no selected habit so I just return nil